lrrbot/stream_status.py

```diff
@@ -33,6 +34,12 @@ def start_check_stream(self):
 	async def check_stream(self):
 		log.debug("Checking stream")
 		data = await twitch.get_info(use_fallback=False)
+		if self.old_title != data['title']:
+			log.debug("Title changed, parsing for show information")
+			self.old_title = data['title']
+			if self.find_show(self, data['title']):
+				self.show_override = None
```

Comment:
`show_override` is also on `self.lrrbot` and you should call `override_show` instead.
```suggestion
				self.lrrbot.override_show(None)
```

I'm not a fan of killing the override on every title change. If ever I think it should happen when the stream goes offline.

Reply:
I was reluctant to put it in offline in case there were connectivity issues and the stream went down momentarily.